main.py

```diff
@@ -2,6 +2,33 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name="George",
+                         last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu",
+                         last_name="Reaves").update(first_name="Keanu",
+                                                    last_name="Reeves")
+
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(name="Scarlett").delete()
```

Comment:
There is an error in this line. The `Actor.objects.filter(name="Scarlett").delete()` line is incorrect because the `Actor` model does not have a `name` field. It should be `first_name` or `last_name` instead. You might want to filter by `first_name="Scarlett"` or `last_name="Keegan"` or `last_name="Johansson"`.